selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -10,25 +10,22 @@
  */
 package org.eclipse.che.selenium.core.utils;
 
-import static java.lang.String.format;
 import static org.openqa.selenium.logging.LogType.BROWSER;
-import static org.openqa.selenium.logging.LogType.CLIENT;
-import static org.openqa.selenium.logging.LogType.DRIVER;
-import static org.openqa.selenium.logging.LogType.PERFORMANCE;
-import static org.openqa.selenium.logging.LogType.PROFILER;
-import static org.openqa.selenium.logging.LogType.SERVER;
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
-import java.util.Collections;
 import java.util.List;
 import org.eclipse.che.selenium.core.SeleniumWebDriver;
 import org.openqa.selenium.logging.LogEntry;
-import org.openqa.selenium.logging.LogType;
-import org.openqa.selenium.logging.Logs;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * read and store browser logs to the test logs for change of logs level
+ *
+ * @see SeleniumWebDriver method "doCreateDriver"
```

Comment:
Use `{@link package.class#member label}` notation to reference method in the docs.